package.json

```diff
@@ -33,6 +33,8 @@
     "opn": "^1.0.0",
     "serve-index": "^1.1.4",
     "serve-static": "^1.4.0",
-    "wiredep": "^2.0.0"
+    "wiredep": "^2.0.0",
+    "gulp-mocha": "~2.0.0",
```

Comment:
gulp-mocha is just some glue to let me easily run mocha tests from gulp. I'm trying to handle non front-end tests for the judge and unit tests, since I can't get the front end tests to work from README directions.